NuGet.Services.Dashboard.Operations/Tasks/DashBoardTasks/AzureManagementTasks/CreateTrafficManagerStatusOverviewReportTask.cs

```diff
@@ -29,36 +29,48 @@ public override void ExecuteCommand()
             request.Headers.Add("x-ms-version: 2014-02-01");
             request.PreAuthenticate = true;
             request.Method = "GET";
-            WebResponse respose = request.GetResponse();
+            WebResponse response = request.GetResponse();
             //Schema of the response would be as specified in http://msdn.microsoft.com/en-us/library/azure/hh758251.aspx
-            Console.WriteLine(respose);
-            using (var reader = new StreamReader(respose.GetResponseStream()))
-            {               
+            using (var reader = new StreamReader(response.GetResponseStream()))
+            {
+                var responseContent = reader.ReadToEnd();
+                Console.WriteLine(responseContent);
+
                 XmlDocument doc = new XmlDocument();
-                doc.LoadXml(reader.ReadToEnd());
+                doc.LoadXml(responseContent);
                 XmlNodeList parentNode = doc.GetElementsByTagName("Endpoint", "http://schemas.microsoft.com/windowsazure");
-                List<Tuple<string, string>> endpointValues = new List<Tuple<string, string>>();
+                //End point Structure
+                //<Endpoint >
+                //    <DomainName>    Name                </DomainName>
+                //    <Status>        Enabled             </Status>
+                //    <Type>          CloudService        </Type>
+                //    <Location>      North Central US    </Location>
+                //    <MonitorStatus> Online              </MonitorStatus>
+                //    <Weight>        1                   </Weight>
+                //</Endpoint>
+
+                List <Tuple<string, string>> endpointValues = new List<Tuple<string, string>>();
                 foreach (XmlNode node in parentNode)
```

Comment:
Maybe rename it to `endpointNode` instead so that it's clear that it's an Endpoint node? Also can you `var` this?